op-node/p2p/cli/load_config.go

```diff
@@ -207,6 +217,30 @@ func loadDiscoveryOpts(conf *p2p.Config, ctx *cli.Context) error {
 	return nil
 }
 
+func resolveURLIP(rawurl string) (string, error) {
```

Comment:
Would be great to add a unit-test with the conduit enode as test-vector.
And then make the `net.LookupIP` function an argument to this helper function, so we can mock the DNS lookup part during testing, to avoid flakes.

Reply:
added